log/formatters.go

```diff
@@ -23,27 +25,67 @@ type jsonFormatter struct {
 	timestampColumn string
 }
 
+// Defining a log line type so that we can use a much faster JSON marshall-ing package
+type logLine map[string]interface{}
+
+// Implementing Marshaler
+func (m logLine) MarshalJSONObject(enc *gojay.Encoder) {
+	for k, v := range m {
+		if asString, valueIsString := v.(string); valueIsString {
```

Comment:
did you use a shitload of `if` statements instead of `switch... case` on purpose?